filebeat/docs/inputs/input-filestream-reader-options.asciidoc

```diff
@@ -141,3 +141,93 @@ The default is 16384.
 
 The maximum number of bytes that a single log message can have. All bytes after
 `mesage_max_bytes` are discarded and not sent. The default is 10MB (10485760).
+
+[float]
+===== `parsers`
+
+This option expects a list of parsers the log line has to go through. 
+
+Avaliable parsers:
+- `multiline`
+- `ndjson`
```

Comment:
This isn't rendering correctly as-is.

```suggestion
Available parsers:

* `multiline`
* `ndjson`
```

Reply:
It doesn't look like this syntax was fixed. See https://www.elastic.co/guide/en/beats/filebeat/master/filebeat-input-filestream.html#_parsers